config/locales/en.yml

```diff
@@ -53,9 +53,24 @@ en:
       scans: 'Scan requests'
   confirmation_email:
     contact:
-      default: "  (650) 723-1493\n  greencirc@stanford.edu"
-    default:
-      from: 'greencirc@stanford.edu'
+      HOOVER:
```

Comment:
I could go either way.  I'm using I18n's default value behavior to fallback but that could easily be done with a hash as well.  I'll migrate them into `application.rb` and push it up momentarily.